xchange-stream-kraken/src/main/java/info/bitrich/xchangestream/kraken/KrakenStreamingExchange.java

```diff
@@ -29,7 +29,7 @@
 
   public KrakenStreamingExchange() {}
 
-  private static String pickUri(boolean isPrivate, boolean useBeta) {
+  public static String pickUri(boolean isPrivate, boolean useBeta) {
```

Comment:
Why make this public?

Reply:
@badgerwithagun I use these urls directly in my own code and would like to expose them.